packages/vm/vmimpl/sandbox.go

```diff
@@ -67,8 +67,13 @@ func (s *contractSandbox) Request() isc.Calldata {
 	return s.reqctx.req
 }
 
+const ReqAssetsBagSizeLimit = 25 // this const should be equal to the one in request.move
+
 func (s *contractSandbox) Send(par isc.RequestParameters) {
 	s.Ctx.GasBurn(gas.BurnCodeSendL1Request, uint64(s.reqctx.numPostedOutputs))
+	if len(par.Assets.Coins)+len(par.Assets.Objects) > ReqAssetsBagSizeLimit {
+		panic(vm.ErrAssetsOversize)
+	}
```

Comment:
What if a request calls Send twice? One with 20 assets, and again with 20.

The request is exceeding the limit but the check will not trigger.